vue-frontend/src/components/ApplyForCareer.vue

```diff
@@ -418,6 +435,10 @@ export default {
         type: "Jobs Posting Website",
         url: location.toString(),
       },
+      disableInput: false,
+      disableButton: false,
+      onButtonLoading: false,
```

Comment:
What is purpose of `disableButton` & `onButtonLoading` ? Why do we need both of them, I think only one flag is enough